content/docs/1.8.0/nodes-and-volumes/volumes/expansion.md

```diff
@@ -125,20 +125,20 @@ If you cannot enable it but still prefer to do online expansion, you can:
 
 #### RWX volume
 
-Longhorn currently does not support fully automatic expansion of the filesystem (NFS) for RWX volumes.  You can expand the filesystem manually using one of the following methods:
+As of v1.8.0, Longhorn does support fully automatic online expansion of the filesystem (NFS) for RWX volumes.  However, the feature requires that not only the v1.8.0 Longhorn Manager and CSI components be running, but also the v1.8.0 version of the Share Manager, which manages the NFS export.  If you have upgraded from a previous version, the Share Manager pods (one for each RWX volume) have not been upgraded automatically, to avoid disruption during the upgrade.  
```

Comment:
```suggestion
From v1.8.0, Longhorn supports fully automatic online expansion of the filesystem (NFS) for RWX volumes.  The feature requires:
 
 - v1.8.0 of Longhorn Manager.
 - that CSI components are running.
 - v1.8.0 of the Share Manager, which manages the NFS export.  
 
 If you have upgraded from a previous version, the Share Manager pods (one for each RWX volume) have not been upgraded automatically, to avoid disruption during the upgrade.  
```

Reply:
Done, with a small tweak.